spec/integ/crypto/verification.spec.ts

```diff
@@ -1343,26 +1348,37 @@ describe("verification", () => {
 
             await sendBackupGossipAndExpectVersion(requestId!, "InvalidSecret", matchingBackupInfo);
 
+            // the backup secret should not be cached
+            const cachedKey = await retrieveBackupPrivateKeyWithDelay();
+            expect(cachedKey).toBeNull();
+        });
+
+        /**
+         * Waits briefly to ensure the secret key has been received before retrieving the backup private key.
+         */
+        async function retrieveBackupPrivateKeyWithDelay() {
             // We are lacking a way to signal that the secret has been received, so we wait a bit..
             jest.useRealTimers();
             await new Promise((resolve) => {
                 setTimeout(resolve, 500);
             });
             jest.useFakeTimers({ doNotFake: ["queueMicrotask"] });
 
-            // the backup secret should not be cached
-            const cachedKey = await aliceClient.getCrypto()!.getSessionBackupPrivateKey();
-            expect(cachedKey).toBeNull();
-        });
+            return aliceClient.getCrypto()!.getSessionBackupPrivateKey();
+        }
 
         /**
          * Common test setup for gossiping secrets.
          * Creates a peer to peer session, sends the secret, mockup the version API, send the secret back from sync, then await for the backup check.
+         * @param expectBackup - The expected result for the key backup request.
```

Comment:
Despite its name, it's not really the expected result. Also by convention we leave a blank line before the first `@param:`


```suggestion
         *
         * @param expectBackup - The result to be returned from the `/room_keys/version` request.
```

Reply:
improve expectBackup param description